src/js/App/Sidenav/Navigation.js

```diff
@@ -183,6 +183,7 @@ export const Navigation = () => {
          * Between chrome 2.0 apps navigation
          */
         !parent?.active && dispatch(clearActive());
+        dispatch(toggleGlobalFilter(false));
```

Comment:
Won't this enable the global filter in other bundles than insights? For example the settings namespace has SPA enabled for multiple apps.

Reply:
Not really, the global filter is hidden behind `activeLocation === 'insights'`, this just sets the enable/disable state. Not the visibility of global filter.